src/logic/playercommand.cc

```diff
@@ -2060,6 +2060,50 @@ void CmdToggleMuteMessages::write(FileWrite& fw, EditorGameBase& egbase, MapObje
 	fw.unsigned_8(all_);
 }
 
+// CmdMarkMapObjectForRemoval
```

Comment:
Superfluous comment

Reply:
We have such a comment at the start of every PlayerCommand implementation so I added it for consistency

Reply:
Consistency is good, so let's keep it for now.